src/appmixer/trello/list/DeleteCard/component.json

```diff
@@ -0,0 +1,75 @@
+{
+    "name": "appmixer.trello.list.DeleteCard",
+    "author": "Appmixer <info@appmixer.com>",
+    "icon": "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjUwMCIgaGVpZ2h0PSIyNTAwIiB2aWV3Qm94PSIwIDAgMjU2IDI1NiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiBwcmVzZXJ2ZUFzcGVjdFJhdGlvPSJ4TWlkWU1pZCI+PGRlZnM+PGxpbmVhckdyYWRpZW50IHgxPSI1MCUiIHkxPSIwJSIgeDI9IjUwJSIgeTI9IjEwMCUiIGlkPSJhIj48c3RvcCBzdG9wLWNvbG9yPSIjMDA5MUU2IiBvZmZzZXQ9IjAlIi8+PHN0b3Agc3RvcC1jb2xvcj0iIzAwNzlCRiIgb2Zmc2V0PSIxMDAlIi8+PC9saW5lYXJHcmFkaWVudD48L2RlZnM+PHJlY3QgZmlsbD0idXJsKCNhKSIgd2lkdGg9IjI1NiIgaGVpZ2h0PSIyNTYiIHJ4PSIyNSIvPjxyZWN0IGZpbGw9IiNGRkYiIHg9IjE0NC42NCIgeT0iMzMuMjgiIHdpZHRoPSI3OC4wOCIgaGVpZ2h0PSIxMTIiIHJ4PSIxMiIvPjxyZWN0IGZpbGw9IiNGRkYiIHg9IjMzLjI4IiB5PSIzMy4yOCIgd2lkdGg9Ijc4LjA4IiBoZWlnaHQ9IjE3NiIgcng9IjEyIi8+PC9zdmc+",
+    "description": "Delete a card.",
+    "private": false,
+    "version": "1.0.0",
+    "auth": {
+        "service": "appmixer:trello"
+    },
+    "quota": {
+        "manager": "appmixer:trello",
+        "resources": "requests",
+        "scope": {
+            "userId": "{{userId}}"
+        }
+    },
+    "inPorts": [
+        {
+            "name": "in",
+            "schema": {
+                "type": "object",
+                "properties": {
+                    "boardId": {
+                        "oneOf": [
+                            { "type": "string" },
+                            { "type": "null" }
+                        ]
+                    },
+                    "boardCardId": { "type": "string" }
+                },
+                "required": [
+                    "boardCardId"
+                ]
+            },
+            "inspector": {
+                "inputs": {
+                    "boardId": {
+                        "type": "select",
+                        "label": "Board",
+                        "index": 1,
+                        "source": {
+                            "url": "/component/appmixer/trello/list/ListBoards?outPort=boards",
+                            "data": {
+                                "transform": "./transformers#boardsToSelectArray"
+                            }
+                        },
+                        "tooltip": "Select a board."
+                    },
+                    "boardCardId": {
+                        "type": "select",
+                        "label": "Card ID",
```

Comment:
in some components there is Card ID, some have `Card`. I think we should use one format: 

label ` Card`
tooltip:  `Select a card or card ID from previous component.`

Reply:
or maybe just `Select a card.` ?

Reply:
the same apply for the Board. Sometimes it's Board, UpdateBoard has the Board ID